### ISSUE TITLE
Local file error when using a remote to get the local file

### ISSUE THREAD

**Issue by travis-crowder-kr** [rhysd/actionlint#40]
If you checkout a repository to a local path and try to use it, linting will fail with a file not found.

Example:
```
- name: Checkout token action
  uses: actions/checkout@v2
  with: 
    repository: qwerty/create-github-server-token
    path: ./.helpers/create-github-server-token

- name: Generate GitHub Token
  uses: ./.helpers/create-github-server-token
  id: create-token
...
```
```
neither action.yaml nor action.yml is found in directory
```

This error is expected because the file does not exist locally; it gets downloaded in the pipeline.

**Comment by rhysd** [comment]
The use case makes sense. When I implemented the check for local action, I did not assume that the action is checked out from somewhere.

I agree that the check should not raise an error when no action is found in the directory.

Related to #25

**Comment by rhysd** [comment]
Please try `-ignore` option meanwhile.

```
actionlint -ignore 'neither action\.yaml nor action\.yml is found in directory'
```

**Comment by rhysd** [comment]
@travis-crowder-kr BTW, what is the reason not to use the action directly?

```yaml
- name: Generate GitHub Token
  uses: qwerty/create-github-server-token
  id: create-token
```

**Comment by rhysd** [comment]
The fix was released at [v1.6.3](https://github.com/rhysd/actionlint/releases/tag/v1.6.3).
